src/structures/GuildChannel.js

```diff
@@ -48,7 +49,7 @@ class GuildChannel extends Channel {
      */
     this.permissionOverwrites = new PermissionOverwriteManager(this);
 
-    this._patch(data);
+    if (data && immediatePatch) this._patch(data);
```

Comment:
you can just pass `immediatePatch` into the super and simply remove this patch call because _patch is called in `channel`

Reply:
Not sure I can. The patch on this was hard-coded to false, I don't want to override that by setting a default here.

Reply:
isn't it the same behaviour both ways? im not crazy?